### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

File: mkdocs/docs/code-samples/knative/canary-fixedsplit.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ kubectl wait ksvc/sample-app --for condition=Ready --timeout=120s
 kustomize build $ITER8/samples/knative/canaryfixedsplit/experimentalservice | kubectl apply -f -
 ```
 
-??? info "Look inside output of `kustomize build $ITER8/.../baseline`"
+??? info "Look inside the output of the `baseline`"
```
Comment (Alan-Cha) [inline mkdocs/docs/code-samples/knative/canary-fixedsplit.md]: Extra attention

Comment (sriumcp) [reply mkdocs/docs/code-samples/knative/canary-fixedsplit.md]: We want to look inside the output of a command... the command is kustomize build (not baseline).

File: mkdocs/docs/code-samples/knative/canary-fixedsplit.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ kustomize build $ITER8/samples/knative/canaryfixedsplit/experimentalservice | ku
                 - name: T_VERSION
     ```
 
-??? info "Look inside output of `kustomize build $ITER8/.../experimentalservice`"
+??? info "Look inside the output of the `candidate`"
```
Comment (sriumcp) [inline mkdocs/docs/code-samples/knative/canary-fixedsplit.md]: same comment as above goes... command is kustomize build (not candidate)

**Review by sriumcp — CHANGES_REQUESTED** [review]
minor changes requested

**Review by sriumcp — DISMISSED** [review]
lgtm

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**